src/github_runner_image_builder/cli.py

```diff
@@ -148,14 +148,14 @@ def get_latest_build_id(cloud_name: str, image_name: str) -> None:
     "Ignored if --experimental-external is not enabled",
 )
 @click.option(
-    "--upload-cloud",
+    "--upload-clouds",
     default="",
-    help="EXPERIMENTAL: Different cloud to use to upload the externally built image. The cloud "
-    "connection parameters should exist in the clouds.yaml. Ignored if --experimental-external is"
-    " not enabled, as a part of external build mode parameter.",
+    help="EXPERIMENTAL: Comma separated list of different clouds to use to upload the externally "
+    "built image. The cloud connection parameters should exist in the clouds.yaml. Ignored if "
+    "--experimental-external is not enabled, as a part of external build mode parameter.",
 )
 # click doesn't yet support dataclasses, hence all arguments are required.
-def run(  # pylint: disable=too-many-arguments
+def run(  # pylint: disable=too-many-arguments, too-many-locals
```

Comment:
Perhaps split this into multiple functions? This linting rule being suppressed indicates that the function is becoming too complex

Reply:
It isn't complex - it only has 2 if/else statements, the variables come from the number of arguments which cannot be reduced due to the limitation of click library (it doesn't allow for argument wrapper datastructures).